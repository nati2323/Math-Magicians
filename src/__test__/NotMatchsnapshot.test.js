import React from 'react';
import renderer from 'react-test-renderer';
import  NotMatch from '../pages/NotMatch'

it('NotMatch renders correctly', () => {
  const tree = renderer.create(
      <NotMatch />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});