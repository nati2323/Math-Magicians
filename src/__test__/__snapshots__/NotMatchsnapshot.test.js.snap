// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotMatch renders correctly 1`] = `
<div>
  <h3>
    No match for this page
  </h3>
</div>
`;
